src/go/rpk/pkg/cli/cmd/redpanda/tune.go

```diff
@@ -214,8 +214,10 @@ func tune(
 
 	results := []result{}
 	includeErr := false
+	allDisabled := false
 	for _, tunerName := range tunerNames {
 		enabled := factory.IsTunerEnabled(tunerName, conf.Rpk)
+		allDisabled = allDisabled || !enabled
```

Comment:
Isn't `allDisabled` going to be `true` even if only one tuner is disabled?

What about:
```
allDisabled := true
...
allDisabled = allDisabled && !enabled
```

Reply:
Yes it is. That's what I get for changing it at the last moment. 🤦🏽